crates/ty_python_semantic/resources/mdtest/import/cyclic.md

```diff
@@ -0,0 +1,79 @@
+## Cyclic imports
+
+### Regression tests
+
+#### Issue 261
+
+See: <https://github.com/astral-sh/ty/issues/261>
+
+`main.py`:
+
+```py
+from foo import bar
+
+reveal_type(bar)  # revealed: <module 'foo.bar'>
+```
+
+`foo/__init__.py`:
+
+```py
+from foo import bar
+
+__all__ = ["bar"]
+```
+
+`foo/bar/__init__.py`:
+
+```py
+# empty
+```
+
+#### Issue 113
+
+See: <https://github.com/astral-sh/ty/issues/113>
```

Comment:
I added a test for https://github.com/astral-sh/ty/issues/113 here as well. The issue is not fixed with this PR (I didn't have a closer look into why), but we now infer `Unknown` instead of `Never`.